MapboxDirectionsTests/V5Tests.swift

```diff
@@ -52,6 +52,13 @@ class V5Tests: XCTestCase {
             XCTAssertEqual(task.state, .completed)
         }
         
+        let opts = route!.routeOptions
+        
+        XCTAssertEqual(opts.profileIdentifier, .automobile)
+        XCTAssertTrue(opts.includesSteps)
+        XCTAssertTrue(opts.includesAlternativeRoutes)
+        XCTAssertEqual(opts.routeShapeResolution, .full)
+        
         XCTAssertNotNil(route)
```

Comment:
Move this assertion up above any assertions that depend on `route` being non-nil.